website/data/users.json

```diff
@@ -103,6 +103,20 @@
         "pinned": true,
         "isMember": true
     },
+    {
```

Comment:
Need to swap IHS Markit for S&P Global above.

Also missing Adaptable & Infrasgistics I think